Update community-plugins.json

kryptonianson opened the pull request:
added obsidian-markdown-tags

# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/binarynoir/obsidian-markdown-tags

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [x]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

kryptonianson commented:
made required adjustments

joethei commented:
[.tags {](https://github.com/binarynoir/obsidian-markdown-tags/blob/c7f754b1361f887b7288948c672a56c4c22d7170/styles.css#L1)
This is a bit generic and could conflict with other plugins.

[const tagSyntaxRegex = /\[\[<?tag\|(?<label>[^\]|]+)(?:\|(?<bgcolor>[^\]|]*))?(?:\|(?<fgcolor>[^\]|]*))?\]\]/g;](https://github.com/binarynoir/obsidian-markdown-tags/blob/c7f754b1361f887b7288948c672a56c4c22d7170/main.ts#L14)
You should use a different syntax, the double brackets are already used in the core app, so this could lead to some conflicts.
Especially in this example from your README: `[[tag|label]]`, which is also a valid Wikilink.

kryptonianson commented:
> [.tags {](https://github.com/binarynoir/obsidian-markdown-tags/blob/c7f754b1361f887b7288948c672a56c4c22d7170/styles.css#L1) This is a bit generic and could conflict with other plugins.
> 
> [const tagSyntaxRegex = /[[<?tag|(?[^\]|]+)(?:|(?[^\]|]_))?(?:|(?[^\]|]_))?]]/g;](https://github.com/binarynoir/obsidian-markdown-tags/blob/c7f754b1361f887b7288948c672a56c4c22d7170/main.ts#L14) You should use a different syntax, the double brackets are already used in the core app, so this could lead to some conflicts. Especially in this example from your README: `[[tag|label]]`, which is also a valid Wikilink.

These were really good observations. I have changed the css styles to .bn-tags and .bn-arrow-tags. In addition I have changed the tag syntax to ((tag|label)) using double '((' instead which is actually more representative of the final render and makes more sense. I hope this is satisfactory. And thank you!
